apps/api/src/app/execution-details/execution-details.controller.ts

```diff
@@ -1,4 +1,4 @@
-import { ClassSerializerInterceptor, Controller, Get, Param, UseGuards, UseInterceptors } from '@nestjs/common';
+import { ClassSerializerInterceptor, Controller, Get, Param, Query, UseGuards, UseInterceptors } from '@nestjs/common';
```

Comment:
```suggestion
import { ClassSerializerInterceptor, Controller, Get, Query, UseGuards, UseInterceptors } from '@nestjs/common';
```
We don't need `Param` anymore.